components/channel_info_rhs/menu.test.tsx

```diff
@@ -4,55 +4,82 @@
 import React from 'react';
 import {fireEvent, screen} from '@testing-library/react';
 
-import {Channel} from 'mattermost-redux/types/channels';
+import {Channel, ChannelStats} from 'mattermost-redux/types/channels';
 import {renderWithIntl} from 'tests/react_testing_utils';
 import Constants from 'utils/constants';
 
 import Menu from './menu';
 
 describe('channel_info_rhs/menu', () => {
+    const defaultProps = {
+        channel: {type: Constants.OPEN_CHANNEL} as Channel,
+        channelStats: {files_count: 3} as ChannelStats,
+        isArchived: false,
+        actions: {
+            openNotificationSettings: jest.fn(),
+            showChannelFiles: jest.fn(),
+        },
+    };
```

Comment:
I like the consolidation of the repeated props! Could you redefine the props actions functions in a `beforeEach`? The calls assertions work out between tests because we don't have a test that expects a call, followed by one that doesn't, but if we did, we'd get a failing test because its called between tests. 0/5 if you redefine just the props actions functions or all of props. As a fairly typical example picked at random: https://github.com/mattermost/mattermost-webapp/blob/master/components/threading/global_threads/thread_list/thread_list.test.tsx#L46-L53